pkg/controller/kogitoapp/resource/kogito_imagestream.go

```diff
@@ -29,32 +28,29 @@ import (
 	imgv1 "github.com/openshift/api/image/v1"
 )
 
-const (
```

Comment:
why removing this block of contants and declare it individually?

Reply:
Goland lint.